site/src/components/Footer.tsx

```diff
@@ -87,38 +83,67 @@ const SOCIALS: { name: string; icon: ReactNode; href: string }[] = [
     icon: <Icon className="fab fa-discord" />,
     href: "https://discord.gg/PefPteFe5j",
   },
+  {
+    name: "GitHub",
+    icon: <Icon className="fab fa-github" />,
+    href: "https://github.com/blockprotocol",
+  },
 ];
 
 const Socials = (
-  <Box mt={3} display="flex">
-    {SOCIALS.map(({ href, icon }) => (
-      <Link
-        href={href}
-        key={href}
-        sx={{
-          "&:not(:first-child)": {
-            marginLeft: 2.5,
-          },
-          ":hover": {
-            svg: {
-              color: (theme) => theme.palette.gray[30],
+  <Box
+    mt={3}
+    display="flex"
+    sx={{
+      flexDirection: {
+        xs: "column",
+        md: "row",
+      },
+      alignItems: { xs: "left", md: "center" },
+    }}
+  >
```

Comment:
My preference would be for the button to wrap to the next line when only when it no longer fits on the same line, like so:

https://user-images.githubusercontent.com/42802102/148071025-f7616df4-ed7e-4fab-b7eb-55d52ab52578.mov

If you agree, the following would accomplish this (in addition to adding `flexShrink: 0` to both flex children):
```suggestion
<Box
    mt={3}
    display="flex"
    flexDirection="row"
    alignItems="center"
    flexWrap="wrap"
  >
```